metrics.yaml

```diff
@@ -0,0 +1,283 @@
+      - rs01-event
+    description: |
+      The origin of the URL associated with the page
+      visit. Calculated by applying `new URL(url).origin`.
+      See the documentation for
+      [url.origin](https://developer.mozilla.org/en-US/docs/Web/API/URL/origin).
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  referrer_origin:
+    type: string
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The origin of the referrer URL for the page
+      loading in the tab.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  title:
+    type: string
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The contents of the title element in the head
+      of the page.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  og_type:
+    type: string
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The `og:type` meta tag contents (e.g. `<meta type="og:type"
+      contents="article" />`).
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  og_description:
+    type: string
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The `og:description` meta tag contents (e.g. `<meta
+      type="og:description" contents="..." />`). If this
+      isn't supplied, then attempts to look at the meta
+      description contents (e.g. `<meta name="description"
+      content="...">`).
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+page.audio:
+  max_relative_scroll_depth:
+    type: quantity
+    unit: percentage
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The largest depth reach on the page, as a
+      proportion of the total page height.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  max_pixel_scroll_depth:
+    type: quantity
+    unit: pixels
```

Comment:
how strict is the unit? Is there any type-checking? Sometimes these scroll metrics seem like they're supposed to be integers but you get a float here and there.